src/SW_Weather.c

```diff
@@ -1065,17 +1350,99 @@ void SW_WTH_setup(void) {
 			w->scale_skyCover[month] = sky;
 			w->scale_wind[month] = wind;
 			w->scale_rH[month] = rH;
+            w->scale_actVapPress[month] = actVP;
+            w->scale_shortWaveRad[month] = shortWaveRad;
 		}
 
 		lineno++;
 	}
 
+    // Check if monthly flags have been chosen to override daily flags
+    w->has_sfcWind = (w->use_windSpeedMonthly) ? swFALSE : w->has_sfcWind;
+    w->has_windComp = (w->use_windSpeedMonthly) ? swFALSE : w->has_windComp;
+
+    w->has_hurs = (w->use_relHumidityMonthly) ? swFALSE : w->has_hurs;
+    w->has_hurs2 = (w->use_relHumidityMonthly) ? swFALSE : w->has_hurs2;

```

Comment:
If a run uses monthly humidity inputs, then all humidity related daily inputs should be turned off (otherwise vp and rH may end up being inconsistent), i.e.,
```
    if (w->use_relHumidityMonthly) {
      w->has_hurs = swFALSE;
      w->has_hurs2 = swFALSE;
      w->has_huss = swFALSE;
      w->has_vp = swFALSE;
    }
```